lib/tutorials/en_US/auth.md

```diff
@@ -64,94 +165,137 @@ const start = async () => {
 
 The last thing we do is tell a route to use the strategy named `simple` for authentication.
 
-## Schemes
-
-A `scheme` is a method with the signature `function (server, options)`. The `server` parameter is a reference to the server the scheme is being added to, while the `options` parameter is the configuration object provided when registering a strategy that uses this scheme.
-
-This method must return an object with *at least* the key `authenticate`. Other optional methods that can be used are `payload` and `response`.
-
-### `authenticate`
-
-The `authenticate` method has a signature of `function (request, h)`, and is the only *required* method in a scheme.
-
-In this context, `request` is the `request` object created by the server. It is the same object that becomes available in a route handler, and is documented in the [API reference](/api#request-object).
-
-`h` is the standard hapi [response toolkit](https://hapijs.com/api#response-toolkit).
-
-When authentication is successful, you must call and return `h.authenticated({ credentials, artifacts })`. `credentials` property is an object representing the authenticated user (or the credentials the user attempted to authenticate with) Additionally, you may also have an `artifacts` key, which can contain any authentication related data that is not part of the user's credentials.
-
-The `credentials` and `artifacts` properties can be accessed later (in a route handler, for example) as part of the `request.auth` object.
+## <a name="cookie" /> hapi-auth-cookie
 
-If authentication is unsuccessful, you can either throw an error or call and return `h.unauthenticated(error, [data])` where `error` is an authentication error and `data` is an optional object containing `credentials` and `artifacts`. There's no difference between calling `return h.unauthenticated(error)` or throwing an error if no `data` object is provided. The specifics of the error passed will affect the behavior. More information can be found in the API documentation for [`server.auth.scheme(name, scheme)`](https://hapijs.com/api#-serverauthschemename-scheme). It is recommend to use [boom](https://github.com/hapijs/boom) for errors.
+[hapi-auth-cookie](https://github.com/hapijs/hapi-auth-cookie) is a plugin that will store a cookie in the users browser once they are authenticated.  This has the option of keeping the user logged in, even after they leave the site.  Here is an example of setting up `hapi-auth-cookie`:
 
-### `payload`
-
-The `payload` method has the signature `function (request, h)`.
+Note:  In this example, we are using the `vision` plugin with `handlebars` to serve our pages.  For more info, please see the [views tutorial](/tutorials/views)
```

Comment:
I'm not sure we should be using `vision` in that part. I feel it could potentially overload newcomers, plus the cookie scheme can be demonstrated without using it.

Reply:
Yeah, I agree...fixing it now